### PR TITLE
[GAP-35] GPU PCI capability proposal

### PR DESCRIPTION
**Pull request by staszek-krotki**
This GAP proposes a specification of GPU capabilities in Provider's offer.

### CODE REVIEW

File: gaps/gap-draft_gpu-pci-capability/gap-draft_gpu-pci-capability-proposal.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+GPU/PCI capability information in the Offer
+
+## Motivation
+This GAP introduces GPU (and possibly other PCI) devices capability description. Providers will specify this information so that Requestors would be able to specify certain processing requirements in Demands.
+
+## Specification
+
+* golem: {
+    * !exp: {
+        * gap-pci: {
```
Comment (pwalski) [inline gaps/gap-draft_gpu-pci-capability/gap-draft_gpu-pci-capability-proposal.md:21]: Do we want to have `gap` in schema? Maybe `pci-device` or `device`, or `devices`?

Comment (staszek-krotki) [reply gaps/gap-draft_gpu-pci-capability/gap-draft_gpu-pci-capability-proposal.md:21]: From my understanding - this is going to be updated to the gap number, once it is assigned, and will stay so as long as it's under experimental namespace.

**Comment by stranger80** [comment]
GAP-35 id assigned. I have also adjusted exp property naming accordingly - please review. 
This is now progressign with experimental implementation, and once stable - we can update the GAP content as well as Golem Standards content.

**Comment by nieznanysprawiciel** [comment]
The implementation of experimental GPU feature is done and GAP was updated accordingly.

**Pull request merged**